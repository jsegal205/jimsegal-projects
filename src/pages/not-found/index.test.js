import React from "react";
import { render, cleanup } from "@testing-library/react";
import NotFound from "./index";

afterEach(cleanup);

describe("NotFound Component", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
  });
});
