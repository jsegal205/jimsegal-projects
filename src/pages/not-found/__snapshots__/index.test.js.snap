// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component renders correctly 1`] = `
<DocumentFragment>
  <h1>
    Not Found
  </h1>
</DocumentFragment>
`;
